### PR TITLE
Prepare release for the stable rps with abstract base class change

### PR DESCRIPTION
**Pull request by Yao725**
Prepare release for #31088  and #31009

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by ArcturusZhang — APPROVED** [review]
Should we also include the information about the class has been changed to abstract along with the accessibility of the constructor?

**Comment by Yao725** [comment]
> Should we also include the information about the class has been changed to abstract along with the accessibility of the constructor?

These breaking changes can be classified into two types, one is we change the class to be abstract and the ctor to be protected, such as Cdn, ContainerRegistey and Network. Another is we only change the ctor to be protected, such as CosmosDB and Servicelinker. That's because we already used customization to support the abstract class, just missed the ctor change. Therefore you will only see the ctor change description in the changelog of these two RPs.

File: sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
-
-### Features Added
+## 1.1.0 (2022-09-13)
```
Comment (ArthurMa1978) [inline sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/CHANGELOG.md:3]: release 1.0.1

Comment (Yao725) [reply sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/CHANGELOG.md:3]: Do you mean that all these 5 RPs' version should be 1.0.1？

**Pull request merged**